src/main/java/de/retest/recheck/ignore/SearchFilterFiles.java

```diff
@@ -56,4 +58,23 @@ private SearchFilterFiles() {}
 			return Collections.emptyList();
 		}
 	}
+
+	public static Map<String, List<Filter>> loadFilterMap() {
```

Comment:
I think, that this should not be part of the api. Sure, loading a filter should be, but collecting the name should not. If we take a look at the MVC-Paradigma and its derivatives, the Model should never know how things are displayed (e.g. color, localization). However, it can be implicitly define things like name if it is part of the Model (e.g. `NamedFilter` - which I would not do).

Either way, Maps are a abstract Containers featuring easy key-value access. If you need the name and must have it here, we should not use a Map if we need access to the key (i.e. its entries), instead a List with a more descriptive Object.

Furthermore, I would not use a Container as such, just load a single `Filter` based on a `Path`, so that one can choose, which filters to load.

So to sum up: If we would have a method `NamedFilter loadNamedFilter(Path)` where we specifically pass the Path, what is hindering us in the GUI to just get the file name (under the assumption that we value the file name as actual name)? **Thus I would just do a method to load a `Filter` and let the GUI decide its name**, unless we decide that the `Filter` should not have its name based on the file name but instead on the file's contents (e.g. a line #name positioning).

Lastly, loading a filter should not be part of `SearchFilterFiles` since the class does not suggest that it can load a filter.